### ISSUE TITLE
TypeScript service warning

### ISSUE THREAD

**Issue by renhongl** [Microsoft/vscode#21281]
- VSCode Version: Code 1.9.1 (f9d0c687ff2ea7aabd85fb9a43129117c0ecf519, 2017-02-09T00:26:45.394Z)
- OS Version: Windows_NT ia32 10.0.10240
- Extensions:

|Extension|Author|Version|
|---|---|---|
|html-snippets|abusaidm|0.0.16|
|theme-atom-one-dark|andischerer|0.0.1|
|vscode-icontheme-nomo-dark|be5invis|1.0.11|
|vscode-eslint|dbaeumer|1.2.5|
|python|donjayamanne|0.3.24|
|vscode-great-icons|emmanuelbeziat|1.1.26|
|vscode-auto-open-markdown-preview|hnw|0.0.3|
|beautify|HookyQR|0.1.10|
|theme-hackershaze|Nuuf|1.1.2|
|new-cmd|odonno|0.2.0|
|seti-icons|qinjia|0.1.3|
|vscode-icons|robertohuertasm|4.0.5|
|theme-obsidian|rprouse|0.2.0|
|background|shalldie|1.1.9|

---

Steps to Reproduce:

1. Sometimes just open one js file, will infinite show this warning.
2. Screen shot: 
![issue](https://cloud.githubusercontent.com/assets/16144833/23263844/64d6f6ee-fa1a-11e6-985d-41055a1355e6.png)

**Comment by roblourens** [comment]
Can you tell me what that means in English?

**Comment by mjbvz** [comment]
@renhongl Are you opening a file on a network drive?

**Comment by mjbvz** [comment]
Two probable causes to this type of issue:

1. You are opening a mapped network drive.  This is fixed in VSCode 1.10 (the current insiders builds)
1. If you see a message about a missing `typescript/...` file, you may be seeing #20496 in which an antivirus program is deleting our TypeScript implementation. See [this comment](https://github.com/Microsoft/vscode/issues/20496#issuecomment-279518845) for workarounds

If neither seems to fit for what you are hitting, please let me know and we can reopen this issue to investigate further

**Comment by renhongl** [comment]
I saw that message about can not find typescript/... path.  My projects did not use typescript, so I go to this path 'C:\Program Files (x86)\Microsoft VS Code\resources\app\extensions' to remove typescript extension. Now vs code working fine. I did not open a mapped network drive, just open vs code, sometimes will happen that warning, that warning information will show then hide,,,,show then hide,,,,all the time.
